guides/hack/04-statements/29-foreach.md

```diff
@@ -1,6 +1,6 @@
 The `foreach` statement iterates over the set of elements in a given collection, starting at the beginning, executing a single statement
-each iteration. On each iteration, the value of the current element is assigned to the corresponding variable, as specified.  The loop body
-is executed zero or more times.  For example:
+each iteration. On each iteration, the value of the current element is assigned to the corresponding variable, as specified. The loop body
```

Comment:
Sorry for formatting changes like this.
vscode does that for me when saving the file.